very_good_core/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 1.0.0 
+
+```
+Note: This release has breaking changes.
+```
```

Comment:
can we do something like this instead?

https://github.com/VeryGoodOpenSource/mockingjay/releases/tag/v2.0.0

we need to tell the developers they should specify the output directory to achieve the previous behavior, otherwise the brick content will be in their current directory